django/test/runner.py

```diff
@@ -711,6 +711,8 @@ def setup_databases(verbosity, interactive, keepdb=False, debug_sql=False, paral
         first_alias = None
         for alias in aliases:
             connection = connections[alias]
+            # make it possible to skip migrations during tests
+            connection.settings_dict['MIGRATE'] = connection.settings_dict['TEST'].get('MIGRATE', True)
```

Comment:
This should probably be done in the application code by setting `DATABASE['TEST']['MIGRATE']` as described in the ticket.